lib/portage/dbapi/bintree.py

```diff
@@ -1234,11 +1234,32 @@ def _populate_local(self, reindex=True):
 
         return pkgindex if update_pkgindex else None
 
+    def _run_trust_helper(self):
+        portage_trust_helper = self.settings.get("PORTAGE_TRUST_HELPER", "true")
+        # getuto is a shell script...
+        ret = os.waitstatus_to_exitcode(os.system(portage_trust_helper))
```

Comment:
New code should use [subprocess.run](https://docs.python.org/3/library/subprocess.html#subprocess.run).

If the command needs to be passed to a shell, pass `shell=True` as a keyword argument.

Reply:
Done